enterprise/frontend/src/metabase-enterprise/license/components/LicenseAndBillingSettings/LicenseAndBillingSettings.tsx

```diff
@@ -67,12 +94,20 @@ const LicenseAndBillingSettings = ({
     );
   }
 
-  const isStoreManagedBilling = MetabaseSettings.isStoreManaged();
-  const token = MetabaseSettings.token();
+  const hasMoreFeaturesAvailable = !areAllFeaturesEnabled(
```

Comment:
We may add more flags in the future (utility or real feature), so this check should be based on a "whitelist" rather than a "blacklist" approach. Otherwise if we add one more flag (say for a new version), then we will end up showing the upsell for all "Pro" subscription running x.42.

Alternatively, we can add a `no-upsell` flag from the backend and control that from there instead.